.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "third-party"]
 	path = third-party
 	url = https://github.com/Wargus/dependencies
+[submodule "StormLib"]
+	path = StormLib
+	url = https://github.com/ladislav-zezula/StormLib
```

Comment:
Possibly
https://github.com/Wargus/StormLib

Reply:
Are you sure? Our fork is far behind.
This branch is [2 commits ahead of](https://github.com/Wargus/StormLib/compare/ladislav-zezula%3AStormLib%3Amaster...master), [330 commits behind](https://github.com/Wargus/StormLib/compare/master...ladislav-zezula%3AStormLib%3Amaster) ladislav-zezula/StormLib:master

Reply:
We have to synchronize it (regularly).